tests/test_backend_tfidf.py

```diff
@@ -33,6 +34,19 @@ def test_tfidf_train(datadir, document_corpus, project):
     assert datadir.join('tfidf-index').size() > 0
 
 
+def test_tfidf_train_input_limited(datadir, document_corpus, project):
```

Comment:
This test is a bit funny...I thik I understand what happens here (the training docs get truncated to 1 character, thus there are no words to train on since the minimum length is 3 characters) but maybe it could be explained in a comment